Fix NavLink onClick errors when disabled (#1453)
sumup-oss/circuit-ui: .changeset/clean-laws-perform.md

---
'@sumup/circuit-ui': patch
---

> Fixed a runtime error occurring when users click on a disabled NavItem with an onClick handler

Reviewer: ```suggestion
Fixed a runtime error that occurred when users click on a disabled item in the `Sidebar` with an `onClick` handler.
```

Author: Still not used to writing these. 🙈
